src/main/java/com/myproject/controller/OrderController.java

```diff
@@ -0,0 +1,103 @@
+package com.myproject.controller;
+
+import com.myproject.entity.Basket;
+import com.myproject.entity.Order;
+import com.myproject.entity.Product;
+import com.myproject.entity.User;
+import com.myproject.entity.enums.OrderStatus;
+import com.myproject.service.BasketService;
+import com.myproject.service.MailService;
+import com.myproject.service.OrderService;
+import com.myproject.util.CheckCodeGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@SessionAttributes({"order"})
+public class OrderController {
+
+    @Autowired
+    private OrderService orderService;
+
+    @Autowired
+    private BasketService basketService;
+
+    @Autowired
+    private MailService mailService;
+
+    @ModelAttribute("order")
+    public Order setupUser() {
+        return new Order();
+    }
+
+    @RequestMapping(value = "/user/create_order", method = RequestMethod.GET)
```

Comment:
optional:
we can use other url, for example: ``/user/order/create``  and  ``user/order/accept``
and then we can move ``/user/order`` over the class